ch02.markdown

```diff
@@ -1934,71 +1935,71 @@ class Point {
 }
 {% endprettify %}
 
-Remember that constructors are not inherited, which means that a
-superclass’s named constructor is not inherited by a subclass. If you
-want a subclass to be created with a named constructor defined in the
-superclass, you must implement that constructor in the subclass.
+Souvenez vous que les constructeurs ne sont pas hérités, ce qui sigifie
+qu'un constructeur nommé d'une classe parente n'est pas hérité par
+sa sous classe. Si vous désirez qu'une sous classe soit créé avec un constructeur nommé
+défini par une super classe, vous devez implémenter ce constructeur dans la sous classe.
 
-#### Invoking a non-default superclass constructor {#invoking-a-non-default-superclass-constructor}
+#### Invoquer un constructeur pas par défaut d'une superclasse {#invoking-a-non-default-superclass-constructor}
 {:.no_toc}
 
-By default, a constructor in a subclass calls the superclass’s unnamed,
-no-argument constructor. If the superclass doesn’t have such a
-constructor, then you must manually call one of the constructors in the
-superclass. Specify the superclass constructor after a colon (`:`), just
-before the constructor body (if any).
+Par défaut, un constructeur d'une sous classe appelle le constructeur non nommé et à
+zéro paramètre de la superclasse.
+Si la superclasse n'a pas ce type de constructeur, alors, vous devez appeler manuellement un des constructeur
```

Comment:
"superclasse" vs  "super-classe" (https://github.com/dartlangfr/dart-up-and-running/commit/dfe1f0acef7bc7832774ae51e36fa5e6db42c15b)
A voir ce qu'on utilse de façon générale dans le fait et reste à faire.